content/en/docs/started/workstation/minikf-gcp.md

```diff
@@ -2,16 +2,6 @@
 title = "Deploy using MiniKF on GCP"
 description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)"
 weight = 40
```

Comment:
```suggestion
title = "Deploy using MiniKF on GCP"
description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)"
weight = 40

+++
```

Reply:
Hello @jbottum! As @alfsuse said, the deployment is failing - take a look at the suggested change that should fix it